lib/tasks/deploy.rake

```diff
@@ -112,4 +132,20 @@ namespace :deploy do
 
     JSON.parse(response.body)
   end
+
+  def process_template(template_file, outfile, hash)
+    renderer = ERB.new(File.new(template_file).read)
+
+    # TODO: replace this hack with ERB#result_with_hash once we upgraded to ruby 2.5 which is
+    # currently not possible as rubocop doesn't support ruby 2.5 yet
```

Comment:
rubocop 0.52 does support ruby 2.5. Or is there something specific that doesn't work?

Reply:
yeah rubocop was not the real problem... not sure what I was thinking